shenyu-sdk/shenyu-sdk-core/src/main/java/org/apache/shenyu/sdk/core/client/AbstractShenyuSdkClient.java

```diff
@@ -62,12 +66,14 @@ public abstract class AbstractShenyuSdkClient implements ShenyuSdkClient {
 
     private final Map<String, List<InstanceEntity>> watcherInstanceRegisterMap = new HashMap<>();
 
+    private final List<ShenyuSdkRequestInterceptor> requestInterceptors = new ArrayList<>();
```

Comment:
thread safety?

Reply:
i think ShenyuSdkClient just init once in ShenyuSdkAutoConfiguration#shenyu-spring-boot-starter-sdk